serveradmin/serverdb/tests/test_ip_addr_type.py

```diff
@@ -27,9 +27,11 @@ class TestIpAddrType(TransactionTestCase):
 
     @classmethod
     def setUpClass(cls):
+        super(TestIpAddrType, cls).setUpClass()
```

Comment:
are the `super()` calls with parameters necessary here? it should already be the default no?

Reply:
You were too late Timo was faster.